Lib/javascript/v8/javascriptrun.swg

```diff
@@ -236,8 +240,10 @@ public:
 };
 
 class SWIGV8_ClientData {
+private:
+  v8::Persistent<v8::FunctionTemplate> clazz_templ;
 public:
-  v8::Persistent<v8::FunctionTemplate> class_templ;
+  size_t index;
```

Comment:
Is this used for anything? I can see anything reading it.

Reply:
It is now. As mentioned, as it's taken into multi-isolate environment, we no longer can save templates' handles to global variables. But we can save index within per-isolate table. Or in other words, `index` effectively replaces `class_templ`. And the private `clazz_templ` is a fall-back for legacy V8 versions.

I've spotted apparent inconsistency in `#if V8_MAJOR_VERSION` condition here. I'll ponder a little bit more about it...

Reply:
Why do we always define `clazz_templ` when it seems it's only needed for V8 < 8?

Can't we use a preprocessor conditional to only have the mechanism needed for the V8 version we're actually compiling against?

Reply:
> Why do we always define `clazz_templ` when it seems it's only needed for V8 < 8?

This seems to be wrong assessment. clazz_templ is used whenever `v8_swig_types` is NULL, but it's not like it's NULL when V8<8.